js/exporter/jspdf/export_data_grid_2.js

```diff
@@ -25,13 +25,15 @@ function exportDataGrid(doc, dataGrid, options) {
 
             pdfGrid.startNewTable(options.drawTableBorder, options.topLeft);
 
+            const { headerStyles, groupStyles, totalStyles } = options;
+            const rowStyles = { headerStyles, groupStyles, totalStyles };
```

Comment:
Destructuring and immediate constructing back looks strange.
How the same scenarios are managed in our asp/mvc/winforms components?